tests/functional/end-to-end.bats

```diff
@@ -381,53 +386,62 @@ test_report_format_impl() {
 	fi
 
 	run valgrind --leak-check=full --show-leak-kinds=all --errors-for-leak-kinds=all --error-exitcode=1 ${SELINT_PATH} -c configs/default.conf -r -s policies/check_triggers
+	echo ${output}
 	[ "$status" -eq 0 ]
 }
 
 @test "nesting_gen_req" {
 	run ${SELINT_PATH} -c configs/default.conf -e W-002 -E -s policies/misc/nesting.*
+	echo ${output}
 	[ "$status" -eq 0 ]
 	count=$(echo ${output} | grep -o "W-002" | wc -l)
 	echo "Status: $status, Count: $count (expected 1)"
-	echo $output
+	echo ${output}
```

Comment:
Shouldn't this and the remaining echo calls in this function just be dropped?  It looks like it's just printing the same output four times in a row.